services/motion/builtin/builtin_test.go

```diff
@@ -214,12 +220,64 @@ func TestMoveSingleComponent(t *testing.T) {
 }
 
 func TestMoveOnMap(t *testing.T) {
-	ms, closeFn := setupMotionServiceFromConfig(t, "../data/wheeled_base.json")
```

Comment:
Does the old test no longer work?

If not why not?

If the old test test does still work, we should preserve it, and add what you've written here as in-addition-to rather than instead-of

Reply:
it does work, will bring it back

Reply:
Excellent.

Do you have a sense of the runtime of this specific test? Both locally on your computer, and on our CI?

Reply:
actually once I examined further it looks like the way this test was originally written no longer works because we removed some imports to solve an import cycle. The third test in TestMoveOnMap has been added to hopefully test something very similar - a trivial straight line path with no obstacles in the way. Is this sufficient to accomplish the goal of the original test?

Reply:
I'm fine with that. What's the runtime of the more elaborate tests on your local computer?

Reply:
3.024s - plan around obstacle
3.607s - full plan + execution around obstacle

Reply:
Ah, so this isn't the 4-minute one.

This should be fine